Gems/GenAIVendorBundle/Code/Include/GenAIVendorBundle/GenAIVendorBundleTypeIds.h

```diff
@@ -19,4 +19,19 @@ namespace GenAIVendorBundle
 
     // Interface TypeIds
     inline constexpr const char* GenAIVendorBundleRequestsTypeId = "{F1D5E6A3-0D13-46F9-8B9B-92A85A145625}";
+
+    // Models TypeIds
+    inline constexpr const char* ClaudeModelConfigurationTypeId = "{5A478AB3-8AC6-4C23-8768-03C2791D6B11}";
+    inline constexpr const char* ClaudeModelMessagesAPITypeId = "{9FFED8C5-A872-4ADF-BC80-74BA1FE92E95}";
+    inline constexpr const char* ClaudeModelTextCompletionsTypeId = "{0310426D-3035-4240-B9E5-D1556AF98B47}";
+    inline constexpr const char* OllamaChatModelConfigurationTypeId = "{CE2D1912-94EE-4BCD-9015-E118900F4051}";
+    inline constexpr const char* OllamaChatModelTypeId = "{E82C0C54-0658-419A-8170-033EF06AEA96}";
+    inline constexpr const char* OllamaModelConfigurationTypeId = "{C5FA34FF-2BBB-4DF4-9AA2-C27320B57F93}";
```

Comment:
```suggestion
    inline constexpr const char* OllamaGenerateModelConfigurationTypeId = "{C5FA34FF-2BBB-4DF4-9AA2-C27320B57F93}";
```

This PR is a good place to add the Generate name to the OllamaModel components. The generate name is used in the ollama documentation as a name in the URL for ollama serve: https://github.com/ollama/ollama?tab=readme-ov-file#rest-api.

Reply:
Or creating a seperate PR.

Reply:
This should be solved in a separate PR. I created an issue out of your comment.